k8s-subst

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script generates K8s yaml files from a set of templates.  To run it you need to 
+# define env vars IMAGE_PATH and IMAGE_TAG such that clearwater images can be found at
+#   IMAGE_PATH/{container name e.g. bono}:IMAGE_TAG
+# The script takes a single parameter which is the path to the Kubernetes directory 
+# (which has the templates subdirectory).
+
+[ -z "$IMAGE_PATH" ] && { echo "Need to set IMAGE_PATH"; exit 1; }
+[ -z "$IMAGE_TAG" ] && { echo "Need to set IMAGE_TAG"; exit 1; }
+[ -z "$1" ] && { echo "Need to specify path to K8s tmpl files"; exit 1; }
+
+TMPL_PATH=$1/templates/*
+
+for f in $TMPL_PATH
+do
+  filename=${f##*/}
+  containername="${filename%.*}" 
+  sed 's@{{IMAGE:\([^}]*\)}}@'$IMAGE_PATH'\/\1\:'$IMAGE_TAG'@g' $1/templates/$containername.tmpl > $1/$containername.yaml
```

Comment:
Bash is terrible. These three lines are just unreadable. 
I'm going to assume they work as expected and tested, because you've used them, but ugh. Bash. 

I don't know if it's quite right to call it 'containername' though, given it's actually going to be e.g. sprout-svc. sprout-depl. 
Not sure how much i care, but would be good if it's possible to make this section more readable? Possibly just a comment on what the steps are actually doing.